br/pkg/restore/log_client/client.go

```diff
@@ -1077,174 +1065,57 @@ func SortMetaKVFiles(files []*backuppb.DataFileInfo) []*backuppb.DataFileInfo {
 	return files
 }
 
-// RestoreAndRewriteMetaKVFiles tries to restore files about meta kv-event from stream-backup.
-func (rc *LogClient) RestoreAndRewriteMetaKVFiles(
-	ctx context.Context,
-	files []*backuppb.DataFileInfo,
-	schemasReplace *stream.SchemasReplace,
-	updateStats func(kvCount uint64, size uint64),
-	progressInc func(),
-) error {
+// validateNoTiFlashReplica makes sure no table contains TiFlash replica
+func (rc *LogClient) validateNoTiFlashReplica() error {
```

Comment:
maybe forget to replace the codes by the function.

Reply:
great observation! did rebase twice and a lot of conflicts, forgot to use the function